t/plugin/grpc-web/server.go

```diff
@@ -18,36 +18,89 @@
 package main
 
 import (
+	"context"
+	"encoding/json"
 	"flag"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
 	"log"
 	"net"
-
-	"apisix.apache.org/plugin/grpc-web/a6"
-	"google.golang.org/grpc"
-	"google.golang.org/grpc/reflection"
+	pb "apisix.apache.org/plugin/grpc-web/a6"
 )
 
-var grpcListenAddress string
+type routeServiceServer struct {
+	savedRoutes []*pb.Route
+}
+
+func (rss *routeServiceServer) GetRoute(ctx context.Context, req *pb.Query) (*pb.Route, error) {
```

Comment:
Would be better to use a backend which responds the data streamingly to test

Reply:
hi, Zexuan, two methods are declared in proto, one is `GetRoute` for unary mode, and the other is `GetRoutes` for server-side streaming mode.